Backpack-SwiftUI/SectionHeader/Classes/BPKSectionHeader.swift

```diff
@@ -0,0 +1,79 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public struct BPKSectionHeader: View {
+    let title: String
+    let description: String?
+    let style: Style
+    let button: BPKButton?
+    let accessibilityLabel: String
```

Comment:
This component does not need an accessibility label.
Right now, the label is applied to each item individually and it will still pass by the title, subtitle and button. But saying "Section header" 3 times. 

Considering
- The title is readable
- The subtitle is readable
- The button is readable, or gets a label

We don't need to provide a separate label.

Reply:
Great note! I actually had that doubt and added it thinking it was needed but will gladly remove it 😄